docs/custom-flows/email-password.mdx

```diff
@@ -690,67 +689,73 @@ In email/password authentication, the sign-in is a process that requires users t
 <Tabs items={["Next.js","React","Remix", "Gatsby", "Expo", "JavaScript"]}>
 
 <Tab>
-<CodeBlockTabs options={['Pages Router', 'App Router']}>
-```tsx filename="pages/sign-in/[[...index]].tsx"
-import { useState } from "react";
+<CodeBlockTabs options={['App Router', 'Pages Router']}>
+```tsx  filename="app/sign-in/[[...sign-in]].page.tsx"
+'use client'
+
+import * as React from "react";
 import { useSignIn } from "@clerk/nextjs";
-import { useRouter } from "next/router";
+import { useRouter } from "next/navigation";
 
 export default function SignInForm() {
   const { isLoaded, signIn, setActive } = useSignIn();
-  const [emailAddress, setEmailAddress] = useState("");
-  const [password, setPassword] = useState("");
+  const [email, setEmail] = React.useState("");
+  const [password, setPassword] = React.useState("");
   const router = useRouter();
-  // start the sign In process.
-  const handleSubmit = async (e) => {
+
+  // Handle the submission of the form
+  const handleSubmit = async (e: React.FormEvent) => {
     e.preventDefault();
     if (!isLoaded) {
       return;
     }
 
+    // Start the SIgn In process
     try {
       const result = await signIn.create({
-        identifier: emailAddress,
+        identifier: email,
         password,
       });
 
       if (result.status === "complete") {
-        console.log(result);
         await setActive({ session: result.createdSessionId });
         router.push("/")
       }
       else {
-        /*Investigate why the login hasn't completed */
+        // If the status is not complete, check why. User may need to 
+        // complete further steps.
         console.log(result);
       }
-
+      // Handle errors
     } catch (err: any) {
       console.error("error", err.errors[0].longMessage)
     }
   };
 
+  // Capture user email and password
   return (
     <div>
-      <form>
+      <form onSubmit={(e) => handleSubmit(e)}>
         <div>
           <label htmlFor="email">Email</label>
-          <input onChange={(e) => setEmailAddress(e.target.value)} id="email" name="email" type="email" />
+          <input onChange={(e) => setEmail(e.target.value)} id="email" name="email" type="email" value={email} />
         </div>
         <div>
           <label htmlFor="password">Password</label>
-          <input onChange={(e) => setPassword(e.target.value)} id="password" name="password" type="password" />
+          <input onChange={(e) => setPassword(e.target.value)} id="password" name="password" type="password" value={password} />
         </div>
-        <button onClick={handleSubmit}>Sign In</button>
+        <button type="submit">Sign In</button>
       </form>
     </div>
   );
 }
 ```
-```tsx filename="app/sign-in/[[...sign-in]].page.tsx"
-"use client"
+
+
+```tsx filename="pages/sign-in/[[...index]].tsx"
```

Comment:
Make sure the comments for App Router example vs Pages Router example are consistent